docs/radar.js

```diff
@@ -295,10 +311,10 @@ function radar_visualization(config) {
     // footer
     radar.append("text")
       .attr("transform", translate(footer_offset.x, footer_offset.y))
-      .text("▲ moved up     ▼ moved down")
+      .text("▲ moved up     ▼ moved down   ★ new    〇 no change")
       .attr("xml:space", "preserve")
       .style("font-family", "Arial, Helvetica")
-      .style("font-size", "10px");
+      .style("font-size", "12px");
```

Comment:
This line causes the merge conflict. The increase of the font size is fine. Just make sure that all `style("font-family", ...` use `config.font_family` instead of the hardcoded value.